server/sonar-db-dao/src/main/java/org/sonar/db/component/ComponentDto.java

```diff
@@ -503,4 +527,9 @@ public ComponentDto copy() {
     copy.createdAt = createdAt;
     return copy;
   }
+
+  // TODO Use it in branch plugin
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1135)